src/readOutputFilter.m

```diff
@@ -0,0 +1,43 @@
+function outputFiltered = readOutputFilter(cfg, filterHeader, filterContent)
```

Comment:
maybe calling it `readAndFilterLogfile` would be clearer, no?

Reply:
Similarly I would go for `columnName` or `nameColumnToFilter` instead of `filterHeader`.

Can't think of something better for `filterContent`.